### ISSUE TITLE
Multiple machineries  

### ISSUE THREAD

**Issue by mohammadrad** [cuckoosandbox/cuckoo#1140]
I was wondering if we could configure cuckoo to use multiple machineries ?

**Comment by jbremer** [comment]
No, this is not possible at the moment, and currently we don't have plans to implement this. You can just deploy multiple Cuckoo setups - in fact, this will be made much easier with #863.

**Comment by jbremer** [comment]
We're currently also investigating what would be the best way to deploy multiple Cuckoo nodes with just a single Web Interface in front of it. But gives us some time for that ;-)

**Comment by mohammadrad** [comment]
That would be great to manage multiple cuckoo nodes with one interface each of which having different machinery.
